bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/thq_4f_ckm_LO_ctcvcp/thq_4f_ckm_LO_ctcvcp_proc_card.dat

```diff
@@ -0,0 +1,10 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model sm
+import model HC_NLO_X0_UFO
+define wdec = u c d s u~ c~ d~ s~ l+ l- vl vl~
+generate p p > x0 t~ b j $$ w+ w-, (t~ > b~ w-, w- > wdec wdec)
+add process p p > x0 t b~ j $$ w+ w-, (t > b w+, w+ > wdec wdec)
```

Comment:
it will not include w > tau since definition of l+ doesn't include tau

Reply:
defined l+ which include tau

Reply:
I guess using decay chain will make this gridpack large, if it's case you can move to madspin for top decay